packages/patternfly-4/react-core/src/components/Chip/Chip.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChipButton from './ChipButton';
+import ChipText from './ChipText';
+import ChipWrapper from './ChipWrapper';
+import { TimesCircleIcon } from '@patternfly/react-icons';
+
+export const ChipVariant = {
+  overflow: 'overflow',
+  closable: 'closable'
+};
+
+const Chip = ({ variant, onClick, children, id, className, ...props }) => {
+  switch (variant) {
+    case ChipVariant.overflow:
+      return (
+        <ChipWrapper isOverflowed className={className} {...props}>
```

Comment:
I don't know if the chip wrapper class is really needed.  Is there a reason we can't have the div here in the Chip Class?  Same Question goes for the ChipText.